coins.yml

```diff
@@ -0,0 +1,212 @@
+- id: 60
+  index: 0x8000003c
+  symbol: ETH
+  trust: ethereum
+  name: Ether
+  link: https://ethereum.org/ether
+  decimals: 18
+  sample: '0xfc10cab6a50a1ab10c56983c80cc82afc6559cf1'
+
+- id: 61
+  index: 0x8000003d
+  symbol: ETC
+  trust: classic
+  name: Ether Classic
+  link: https://ethereumclassic.github.io
+  decimals: 18
+  sample: '0xf3524415b6D873205B4c3Cda783527b2aC4daAA9'
+
+- id: 74
+  index: 0x8000004a
+  symbol: ICX
+  trust: icon
+  name: ICON
+  link: https://icon.foundation/
+  decimals: 18
+  sample: 'hxee691e7bccc4eb11fee922896e9f51490e62b12e'
+
+- id: 118
+  index: 0x80000076
+  symbol: ATOM
+  trust: cosmos
+  name: Cosmos ATOM
+  link: https://cosmos.network/
+  decimals: 6
+  sample: 'cosmos1rw62phusuv9vzraezr55k0vsqssvz6ed52zyrl'
+
+- id: 144
```

Comment:
did you consider calling id as `slip44` and then rename `trust` to `id` ?

Reply:
`id` isn't a good name imo, because it gets referenced as `handler` in the rest of the code. So how about either `trust`, `trustwallet` or `handler`.

Reply:
Renamed to `handler`

Reply:
Renamed to `handler`